wqflask/wqflask/correlation/show_corr_results.py

```diff
@@ -74,6 +74,193 @@ def set_template_vars(start_vars, correlation_data):
     return correlation_data
 
 
+def apply_filters(trait, target_trait, target_dataset, **filters):
+    def __p_val_filter__(p_lower, p_upper):
+        return not (float(trait.get('corr_coefficient', 0.0)) >= p_lower and
+                    float(trait.get('corr_coefficient', 0.0)) <= p_upper)
```

Comment:
A more pythonic (and readable way) to write this is:

```suggestion
        return not (float(p_upper >= trait.get('corr_coefficient', 0.0)) >= p_lower)
```

Reply:
Awesome